_layouts/contact.html

```diff
@@ -1,10 +1,15 @@
 {% include header.html %}
 <section id="main" class="container">
   <header>
-    {{content}}
+        {{content}}
+        <hr>
+        <h4><b>{{page.street}}</b></h4>
+        <h1><b>{% if {{page.lang}} == "FR" %}Téléphone:{% else %}Phone:{% endif %} </b> {{page.phone}}</h1>
```

Comment:
not sure to understand....
veux tu que je fasse : if(french)...texte francais, else... texte anglais juste une fois au lieu de le répéter 3 fois ?